canonical/juju-dashboard
# WD-5218 - Fetch audit logs

## Done

- Fetch audit logs in the table component.
- Create a reusable spinner component.

## QA

- Set your config.local.js to use staging.
- Open `src/pages/Logs/Logs.tsx`.
- Add `<AuditLogsTable />` below the header.
- Load the app and go to the top level logs page.
- You should see the logs that get fetched from JIMM.

## Details

https://warthogs.atlassian.net/browse/WD-5218
